server/src/core/config.rs

```diff
@@ -1010,7 +1010,17 @@ fn merge_all_workspaces(
     let mut merged_raw_config: HashMap<String, ConfigEntryRaw> = HashMap::new();
 
     for workspace_config in workspace_configs {
-        for (key, raw_entry) in workspace_config {
+        for (key,mut raw_entry) in workspace_config {
+
+            match raw_entry.extends.clone() {
+                Some(v)=>{
+                    merged_raw_config.values_mut().for_each(|ra|{update_python_path(&mut raw_entry,ra, key.clone())});
+                },
+                None => {
+                    merged_raw_config.values_mut().for_each(|ra|{update_python_path(ra,&mut raw_entry, key.clone())});
+                }
+            }
+
```

Comment:
I am not sure I 100% follow what you are attempting to do. 

As far as I can understand, you are trying to apply the extends behavior across workspace folders? 

Currently, in our design, extends is essentially just being used for the path hierarchy, so vertically, but not horizontally between workspaces. 

If we were to add it accross workspaces, then we would need a mechanism that is more complete such that it affects all parameters and not just python_path.